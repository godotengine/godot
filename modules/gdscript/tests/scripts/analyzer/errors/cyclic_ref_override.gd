func test():
	print(v)

var v := InnerA.new().f()

class InnerA:
	func f(p := InnerB.new().f()) -> int:
		return 1

class InnerB extends InnerA:
	func f(p := 1) -> int:
		return super.f()
