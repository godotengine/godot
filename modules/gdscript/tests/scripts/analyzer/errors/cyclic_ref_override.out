GDTEST_ANALYZER_ERROR
>> ERROR at line 11: Could not resolve member "f": Cyclic reference.
